helm/distributed-idol/templates/dih/ingress.yaml

```diff
@@ -21,26 +21,11 @@ spec:
   rules:
   - http:
       paths:
-      - path: {{ include "idol-library.ingress.path" (dict "ingress" $component.ingress "path" "/dih/") }}
-        pathType: {{ $pathType }}
-        backend:
-          service:
-            name: {{ $component.name }}
-            port:
-              name: aci-port
       - path: {{ include "idol-library.ingress.path" (dict "ingress" $component.ingress "path" "/index/") }}
         pathType: {{ $pathType }}
         backend:
           service:
             name: {{ $root.Values.indexserviceName }}
             port: 
               name: index-port
```

Comment:
think this was meant to be deleted too?

Reply:
ah I kept that because I thought `$root.Values.indexserviceName` was different to `$component.name`?